### DISCUSSION TITLE
polsatnews - video problem

### DISCUSSION THREAD

**Discussion by AroKol78** [MajkiIT/polish-ads-filter#22368] [Q&A]
#22369
(coś nie działa przenoszenie dyskusji do zgłoszenia)

**Comment by hawkeye116477** [comment]
Jeżeli chodzi o przycisk `convert to Discussion` to jest to raczej dostępne jedynie dla uprawnionych osób. Natomiast twój link normalnie działa...

**Reply by AroKol78** [reply]
chodziło mi o "Create issue from discussion", myślałem, że weźmie tytuł i zawartość z dyskusji i zamieni  w `issues`.

<details><summary>...</summary>
utworzyłem wpierw dyskusje z powodu błędu i problemu z video, jednak potem okazało się, że wina nie leży po stronie błędu, więc chciałem przenieść do zgłoszenia i tym sposobem mam 2x o `polsatnews`. ów błąd:

```
Warning: Cross-Origin Request Blocked: The Same Origin Policy disallows reading the remote resource at https://ipla.pluscdn.pl/dituel/cppolska/nq/nq[...]b6.mp4. (Reason: CORS header ‘Access-Control-Allow-Origin’ does not match ‘https://www.polsatnews.pl’).
```

</details>

**Reply by hawkeye116477** [reply]
https://docs.github.com/en/issues/tracking-your-work-with-issues/creating-an-issue#creating-an-issue-from-discussion

Powinno niby dodać zawartość, ale tytuł sam musisz dobrać. Jednakże jeżeli są szablony do zgłoszeń uruchomione i jest zakaz otwierania pustych zgłoszeń, to wtedy ten przycisk jest jakby bez sensu, bo nie będzie działał dla zwykłych użytkowników (mod albo admin może zrobić puste zgłoszenie bez względu na ustawienia), bo nie wiadomo w które pole ma wstawić. Tak więc panowie czy tam panie z ekipy GH powinni schować ten przycisk dla zwykłych użytkowników jeżeli w konfiguracji jest wyłączone robienie pustych zgłoszeń.
W takim też przypadku jedynie admin lub mod repozytorium może zamienić dyskusję na zgłoszenie.
